Metro-Records/la-metro-dashboard
# Update staging db url

## Overview

We upgraded the staging DB to perform load tests and test out connection pooling. See Metro-Records/la-metro-councilmatic#1065.

### Checklist


## Testing Instructions

 * Run `blackbox_postdeploy`
 * Verify that `LA_METRO_DATABASE_URL` in `configs/staging/.env` has been updated

Handles #127
